BugBountyProgram.md

```diff
@@ -1,8 +1,132 @@
+|-----|------|--------|-------------|
+|$500 |$1,000  |$4,000  |Up to $15,000|
+
+Your report will be assessed by the Boson Protocol team and scored using the Common Vulnerability Scoring Scheme (CVSS).
+
+The critical asset bounty will only be awarded where there is a serious and potentially permanent impact to a user's funds, or to the core protocol itself.
+
+The bug bounty program considers a number of variables in determining rewards. Determinations of eligibility, score and all terms related to an award are at the sole and final discretion of Boson Protocol. 
+
+Issues that have already been submitted by another user or are already known to Boson Protocol are not eligible for bounty rewards.
+
+## SLA
+
+Boson Protocol will make a best effort to meet the following SLAs for security researchers participating in our program:
+
+* Time to first response (from report submission) - 3 business days
+* Time to triage (from report submission) - 5 business days
+* Time to bounty (from triage) - 30 business days
+
+We’ll try to keep you informed about our progress throughout the process.
+
+## Participation Requirements 
+
+Participation in the Boson Protocol Bug Bounty program requires you to adhere to “Responsible Disclosure”. Responsible Disclosure includes:
+
+*  Providing a reasonable amount of time to fix a vulnerability prior to sharing details of the said vulnerability with any other party.
+*  Make a good faith effort to avoid privacy violations, destruction of data, and interruption or degradation of our service. Only interact with accounts you own or with the explicit permission of the account holder.
+*  Not defrauding Boson Protocol users, partners or Boson Protocol itself in the process of participating.
+*  Not profiting from or allowing any other party to profit from a vulnerability outside of the payouts made by this program.
+*  Reporting vulnerabilities with no conditions, demands, or ransom threats.
+Social Engineering attacks against Boson Protocol contributors is deemed a violation with respect to this program. Researchers engaging in Social Engineering attacks against Boson Protocol contributors will be banned from this program. We define Social Engineering as acts that influence people to perform security-impacting actions or divulge confidential information.
+*  Asking our permission before disclosing the vulnerability.
 
-Thank you for helping build dCommerce.
+## How to submit your vulnerability
+
+Please provide us with the following information when submitting a bug to this program:
+
+*  Summary of the bug
+*  Severity of the bug
+*  Steps to reproduce
+*  Working proof of concept + any support materials (code, screenshots, logs etc)
+*  Is any private personal data exposed?
+*  Is any partner data exposed
+*  Are user or partner funds at risk of being lost or irretrievable?
+*  Optional: recommendation to address the bug.
+
+Email your report to (security@bosonprotocol.io)[mailto:security@bosonprotocol.io]

```

Comment:
I am guessing that the backets are the wrong way round, hence why it isn't rendering properly